.azure-pipelines/azure-pipelines.yml

```diff
@@ -116,6 +116,8 @@ stages:
       - template: templates/matrix.yml  # context/controller (ansible-test container management)
         parameters:
           targets:
+            - name: Alpine 3.18
+              test: alpine/3.18
```

Comment:
I'd move this below 3.17. It won't matter in the end, since we'll be removing 3.17, but it's still good to keep things ordered.